src/types.ts

```diff
@@ -87,13 +87,15 @@ export interface VitePWAOptions {
    *
    * `inline` - inject a simple register, inlined with the generated html
    *
-   * `script` - inject <script/> in <head>, with the `sr` to a generated simple register
+   * `script` - inject `<script/>` in `<head>`, with the `sr` to a generated simple register
```

Comment:
replace this with:
`script` - inject `<script/>` in `<head>` with `src` attribute to a generated script to register the service worker

apply same change in L92 below for `script-defer`